libs/model/src/models/vote.ts

```diff
@@ -29,11 +29,11 @@ export default (
     'Vote',
     {
       id: { type: dataTypes.INTEGER, autoIncrement: true, primaryKey: true },
-      poll_id: { type: dataTypes.INTEGER, allowNull: false },
+      poll_id: { type: dataTypes.INTEGER, allowNull: true },
```

Comment:
Another place where `allowNull` should remain `false`.

Reply:
added to migration